autoload/test/go/ginkgo.vim

```diff
@@ -0,0 +1,50 @@
+if !exists('g:test#go#ginkgo#file_pattern')
+  let g:test#go#ginkgo#file_pattern = '\v[^_].*test\.go$'
+endif
+let test#go#ginkgo#patterns = {
+  \ 'test': ['\v^\s*It\("(.*)",', '\v^\s*Context\("(.*)",', '\v.*Describe\("(.*)",'],
+  \ 'namespace': [],
+\}
+
+function! test#go#ginkgo#test_file(file) abort
+  if fnamemodify(a:file, ':t') =~# g:test#go#ginkgo#file_pattern
+    if exists('g:test#go#runner')
```

Comment:
The mantra for this plugin is "zero configuration required" so ideally users don't have to set a runner variable. What if ginkgo file matching search for "github.com/onsi/ginkgo" and the existing go runner file pattern only worked for files that don't contain it?